<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddHomeworkDialog</class>
 <widget class="QDialog" name="AddHomeworkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加作业</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>作业标题:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editTitle"/>
   </item>
   <item>
    <widget class="QLabel" name="label_course">
     <property name="text">
      <string>关联课程:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboCourses"/>
   </item>
   <item>
    <widget class="QLabel" name="label_deadline">
     <property name="text">
      <string>截止时间:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="editDeadline">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="dateTime">
      <datetime>2024-06-30T00:00:00</datetime>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddHomeworkDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddHomeworkDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>